src/main/java/org/apache/commons/io/input/ReversedLinesFileReader.java

```diff
@@ -134,17 +133,35 @@ public ReversedLinesFileReader(final File file, final int blockSize, final Chars
         // Open file
         randomAccessFile = new RandomAccessFile(file, "r");
         totalByteLength = randomAccessFile.length();
-        int lastBlockLength = (int) (totalByteLength % blockSize);
-        if (lastBlockLength > 0) {
-            totalBlockCount = totalByteLength / blockSize + 1;
+        seek (totalByteLength);
+
+    }
+    /**
```

Comment:
All new public and protected method should have complete Javadoc. Here the first sentence is missing as is the since tag.